uber/site_sections/api_access.py

```diff
@@ -0,0 +1,66 @@
+from inspect import getargspec, getmembers, ismethod
+from uber.common import *
+
+
+@all_renderable(*c.API_ACCESS.keys())
+class Root:
+
+    def index(self, session, show_revoked=False, message='', **params):
+        admin_account = session.current_admin_account()
+        api_tokens = session.query(ApiToken)
+        if c.ADMIN not in admin_account.access_ints:
+            api_tokens = api_tokens.filter_by(admin_account_id=admin_account.id)
+        if not show_revoked:
+            api_tokens = api_tokens.filter(ApiToken.revoked_time == None)
+        api_tokens = api_tokens.options(
+            subqueryload(ApiToken.admin_account)
+            .subqueryload(AdminAccount.attendee)) \
+            .order_by(ApiToken.issued_time).all()
+        return {
+            'message': message,
+            'admin_account': admin_account,
+            'api_tokens': api_tokens,
+            'show_revoked': show_revoked,
+        }
+
+    def info(self, session):
+        from uber.decorators import get_innermost
```

Comment:
I don't believe this import is necessary, since everything in ``uber.decorators`` gets pulled in by the ``from uber.common import *`` at the top of this file.